stdlib/private/StdlibUnittest/StdlibUnittest.swift

```diff
@@ -749,12 +756,23 @@ func _stdlib_getline() -> String? {
 func _printDebuggingAdvice(_ fullTestName: String) {
   print("To debug, run:")
   var invocation = [CommandLine.arguments[0]]
+#if os(Windows)
+  var buffer: UnsafeMutablePointer<CChar>?
+  var length: Int = 0
+  if _dupenv_s(&buffer, &length, "SWIFT_INTERPRETER") != 0 {
+    if buffer != nil {
```

Comment:
Very minor drive-by code-style: this might be cleaner as `if let buffer = buffer`. The two `if` conditions could also be folded together:

```
if _dupenv_s(&buffer, &length, "SWIFT_INTERPRETER") != 0, let buffer = buffer {
```